### ISSUE TITLE
Preferred name from incorrect CDB file

### ISSUE THREAD

**Issue by sandertan** [CogStack/MedCATtrainer#31]
Hi @tomolopolis another question and/or bug: I've been cleaning up the concept table with Dutch words that we use for creating the MedCAT CDB file. I noticed that when I use a new CDB file, for a new or existing project, that the "Concept Summary" can still display the "Preferred name" of the concept from a difference CDB file. I'm not experienced in Vue so I haven't been able to pinpoint the root of this issue, but I think this is a bug.

### Example
In the last screenshot, the preferred name of "pneumothorax" contains "NAO". This is a suffix in some names from Dutch MedDRA, and not useful for entity linking, so I removed this from the concept table and generated a new CDB file. I tested it in a Jupyter notebook with MedCAT, and the issue seems resolved: 
![image](https://user-images.githubusercontent.com/9624990/112347711-62dd8f80-8cc7-11eb-847b-996dd8c358c7.png)

Also, in this new CDB I've added a new concept, "methotrexaat", which was not present in the previous CDB. (I still need to add a TUI to this concept so dont worry about that).

In MedCATtrainer, the new concept  "methotrexaat" is correctly identified, so I'm sure the updated CDB is in use. But the preferred name still contains "NAO". I suspect this name is retrieved from another CDB file in the same MedCATtrainer instance.
![image](https://user-images.githubusercontent.com/9624990/112348590-40984180-8cc8-11eb-8f36-c7d457fe3989.png)

**Comment by sandertan** [comment]
Seems to be caused around here: https://github.com/CogStack/MedCATtrainer/blob/a6745a781907f1452be2c9bf7f5176f8da91b2a2/webapp/api/api/utils.py#L60 I'll try to debug it.

**Comment by sandertan** [comment]
I think the application uses a general CUI lookup table that spans across projects, because the GET request for filling in this "Concept Summary" can only pass the CUI, not the ConceptDB ID: `/api/concepts/?cui=C0032326`

Do you think it would make sense to extract this data from the project-specific CDB instead? In our current usecase we're experimenting with different concept databases with different preferred names for CUIs.

**Comment by tomolopolis** [comment]
yes you're correct in thinking the concepts table has a primary key of the the concept cui from a given CDB. The original intention around this was to have potentially many projects using one CDBs worth of concepts, and therefore not forcing folks to import concepts from each CDB per project, but this has been limiting at times. 

We could look to improve the concepts table somehow or the concept pretty name lookup could be improved to alternatively look within the project specific CDB.

**Comment by sandertan** [comment]
Let me know if you would like me to look into adding this change. I'm not sure though if other parts of the application rely on this "one concept table" paradigm as well.

**Comment by sandertan** [comment]
@tomolopolis We can close this one for now. When using different CDB universes with different pretty names, this issue can be solved by setting up multiple MedCATTrainer instances.
